### PR TITLE
Neo4j reader

### PR DESCRIPTION
**Pull request by coyotebush**
Work in progress. Since I'm completely new to Spring, I don't know whether I'm doing anything correctly, but I'm roughly following the [Hubway code](https://github.com/jexp/hubway-sdn).

I'm not even sure how to try it out -- what needs to be done to set up Spring and then call `importStations`, for instance?

If the parsing tweaks look good, I could create another pull request to go ahead and merge just those.

### CODE REVIEW

**Comment by coyotebush** [comment]
While we're at it... probably the Bike/Station/Trip classes should be under a "graph" package or something, to avoid possible conflict with similar classes on the RDBMS side?

File: graph-db-driver/src/main/java/se/kth/csc/moderndb/cbexplorer/parser/TripReader.java [hunk 1]
```diff
@@ -1,7 +0,0 @@
-package se.kth.csc.moderndb.cbexplorer.parser;
-
-/**
- * Created by mhotan on 4/13/14.
- */
-public class TripReader {
```
Comment (mhotan) [inline graph-db-driver/src/main/java/se/kth/csc/moderndb/cbexplorer/parser/TripReader.java:6]: I was thinking the TripReader class was going to be a Spring Controller like class.  Then we could Autowire A GraphService class.  But I think we have to look into how we will define the applicationContext.xml (Spring thing) for this particular Restful service.  The Application Context will define how things are auto wired.

**Comment by mhotan** [comment]
Nothing stands out to me.  Its going to be a lot of experimentation to figure out how to make this work with the framework.

**Comment by coyotebush** [comment]
Looking good, thanks! The import does run quite slowly -- I'm seeing somewhere ~10 trips/second. I wonder if explicitly controlling transactions might affect performance?

I think we should probably merge this soon and leave the REST API for next?

**Comment by mhotan** [comment]
Yeah we can merge this and do REST API and make the import faster next.

**Pull request merged**